### PR TITLE
Add method source example

### PR DESCRIPTION
**Pull request by igorwojda**

### CODE REVIEW

File: instrumentation/sample/src/test/kotlin/de/mannodermaus/sample/ExampleKotlinTest.kt [hunk 1]
```diff
@@ -100,6 +103,12 @@ class ExampleKotlinTest {
         Executable { assertNotNull(value) },
         Executable { assertEquals(15, value.length) })
   }
+  
+  @ParameterizedTest(name = "Upper case for {0}")
+  @MethodSource("getNames")
+  fun parameterizedMethodTest (names: Pair<String, String>) {
+      assertEquals(names.first.toUpperCase(), names.second)
```
Comment (mannodermaus) [inline instrumentation/sample/src/test/kotlin/de/mannodermaus/sample/ExampleKotlinTest.kt:110]: `names.first.toUpperCase()` produces the actual result, `names.second` is the expected outcome. Please switch the param order here, as `assertEquals()` takes the expected one first!

**Review by mannodermaus — CHANGES_REQUESTED** [review]
I don't have a strong preference for adding parameterized tests with other sources to the sample, but I also don't see why we couldn't extend the repertoire. Thanks for the contribution! Added a comment about the parameter order.

File: instrumentation/sample/src/test/kotlin/de/mannodermaus/sample/ExampleKotlinTest.kt (hunk 1 above)
Comment (igorwojda) [reply instrumentation/sample/src/test/kotlin/de/mannodermaus/sample/ExampleKotlinTest.kt]: DOne

**Review by mannodermaus — APPROVED** [review]
Apologies for the delay. Thanks for this!

**Pull request merged**